triage_bot/lib/engine.dart

```diff
@@ -0,0 +1,1499 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
Personally I feel such a long file is difficult to review. Is there a style guide for file of such a length? I do not find any recommendation in https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo.

I see some classes/logics could be separated into their own lib files, like GitHub related classes.

Reply:
I'm very open to suggestions but it's not clear to me what would make things better. The whole file is GitHub-related. :-)